instrumentation/activities/activity.go

```diff
@@ -12,3 +13,7 @@ func NewActivity(userID, name string) activity.Activity {
 		OccuredAt: time.Now(),
 	}
 }
+
+func Identifier(comps ...string) string {
```

Comment:
This seems kind of use-case-specific, maybe doesn't belong in the library itself?

Reply:
My intention was to provide a consistent way of generating IDs for activities. I agree that it's not very...sophisticated. I'm open to alternative suggestions. Just pushing it on to the consumer doesn't feel right to me.